### ISSUE TITLE
Umbraco Forms contains two namespaces for MVC

### ISSUE THREAD

**Issue by bjarnef** [umbraco/Umbraco.Forms.Issues#488]
In Umbraco Forms v8.6.0 I noticed UmbracoForms.Core contains two namespaces for "MVC".

**Umbraco.Forms.MVC** and **Umbraco.Forms.Mvc**

It seems a bit strange that these doesn't share namespace?

![image](https://user-images.githubusercontent.com/2919859/108356269-3a580680-71ec-11eb-9003-245bc5338be2.png)

It seems `Umbraco.Forms.MVC` only contains the `Umbraco.Forms.MVC.Extensions.JQueryUIDatePickerHelper` class.
Is there a reason this doesn't exists in `Umbraco.Forms.Mvc` namespace?

Furthermore I notcied `Umbraco.Forms.Core.Configuration.GetSetting()` is marked as deprecated and we should use `FacadeConfiguration`. How do we access this? I doesn't seem it is documented and so far I have only found `Umbraco.Forms.Core.IFacadeConfiguration`.

![image](https://user-images.githubusercontent.com/2919859/108356657-af2b4080-71ec-11eb-8565-d36f0a24288b.png)

**Comment by bjarnef** [comment]
The current way:

```
var test = Umbraco.Forms.Core.Configuration.GetSetting("myCustomSetting");
```

New?
```
var configuration = Umbraco.Web.Composing.Current.Factory.GetInstance<Umbraco.Forms.Core.IFacadeConfiguration>();
var test = configuration.GetSetting("myCustomSetting");
```

**Comment by bjarnef** [comment]
Okay, regarding the `IFacadeConfiguration` it seems we can inject this in the controller in newer versions of Umbraco Forms. Not sure about the approach in views though?
https://github.com/shaishavkarnani/RecaptchaV3/pull/2/files

Maybe the obsolete message should say "Use IFacadeConfiguration" instead?

**Comment by nul800sebastiaan** [comment]
Thanks for reporting @bjarnef - our lovely legacy codebase is sometimes a bit nonsensical. We'll have a look at it.

**Comment by nul800sebastiaan** [comment]
Not sure if you're still struggling with this or found the workaround? For now "just" use the wrong namespace I guess?

**Comment by bjarnef** [comment]
@nul800sebastiaan the easiest would probably be to update for namespace for `JQueryUIDatePickerHelper` to be inside `Umbraco.Forms.Mvc`, but maybe I breaking change if this helper is something used in form themes.

Regarding `IFacadeConfiguration` I found that I could inject this in the constructor of the custom field type.
In the view I guess we need to use the following:

```
var configuration = Umbraco.Web.Composing.Current.Factory.GetInstance<Umbraco.Forms.Core.IFacadeConfiguration>();
var test = configuration.GetSetting("myCustomSetting");
```

Would be great with some updated documentation regarding the new way to get a custom field setting.

**Comment by warrenbuckley** [comment]
Namespace fixed in upcoming 8.6.2 release with this PR
